scripts/sanity_chk/sanitylib.py

```diff
@@ -2212,6 +2212,7 @@ def __init__(self, board_root_list=[], testcase_roots=[], outdir=None):
         self.start_time = 0
         self.duration = 0
         self.warnings = 0
+        self.testcase_files = {'filename1': 'sample.yaml', 'filename2': 'testcase.yaml'}
```

Comment:
should be class variables, i.e. SAMPLE_FILENAME, TESTCASE_FILENAME and access via self.SAMPLE_FILENAME and self.TESTCASE_FILENAME.

Reply:
Done.